samples/bluetooth/direction_finding_connectionless_rx/src/main.c

```diff
@@ -14,13 +14,17 @@
 #define DEVICE_NAME CONFIG_BT_DEVICE_NAME
```

Comment:
Typo in commit message, shouldn't it be:  `Fix too early per sync create timeout` ?

Reply:
yes, corrected.